### DISCUSSION TITLE
Vertical scrollbar not responding to mouse on Gnome+Wayland

### DISCUSSION THREAD

**Discussion by kvibber** [tsujan/FeatherPad#776] [Q&A]
When running under a GNOME 45 Wayland session on Fedora 39, Featherpad's vertical scroll bar only responds to clicks or dragging within a very narrow range, basically the left edge of the scrollbar and a pixel or two to the right.

The wheel and keyboard scrolling work as expected, as does the horizontal scrollbar.

It works normally when run under KDE Plasma on Wayland, or any X11 desktop session including Gnome!

**Comment by tsujan** [comment]
Since there's nothing about that in FeatherPad's code, the cause should be somewhere else.

I've used FeatherPad under Wayfire, and I'm using it now under LabWC (+LXQt), with the latest stable Qt6 and other packages from Arch: the scrollbar behavior is normal.

I'm converting this to discussion...
